test/edu/csus/ecs/pc2/core/util/AbstractTestCase.java

```diff
@@ -792,6 +792,22 @@ public void editFile(String filename) {
         }
     }
     
+    public void editFile(String filename, String comment) {
```

Comment:
This adds a method which is never called from anywhere in the project.  What's the point in that?